pootle/apps/pootle_app/forms.py

```diff
@@ -55,7 +55,7 @@ class ProjectForm(forms.ModelForm):
     source_language = forms.ModelChoiceField(label=_('Source Language'),
                                              queryset=Language.objects.none())
     fs_plugin = forms.ChoiceField(
-        label=_('FS Plugin'),
+        label=_('Filesystem backend'),
```

Comment:
this isnt actually displayed anywhere - not sure if we can just remove it

Reply:
I think you are right. I will adjust the PR.